packages/dialog/src/web/dialog-web.tsx

```
@@ -0,0 +1,76 @@
+const Title = (() => {
+  if (process.env.NODE_ENV === 'development') {
+    console.log('`Description` from @rn-primitives/dialog/web is only supported on web.');
+  }
+  return null;
+}) as unknown as typeof DialogTitle;
+
+const useRootContext = () => {
+  throw new Error(
+    'Cannot access the web useRootContext on a web platform. Please import from `@rn-primitives/dialog` or `@rn-primitives/dialog/web`'
+  );
+  return {} as RootContextReturnType;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)